Core/HLE/proAdhoc.cpp

```diff
@@ -82,32 +87,62 @@ int isLocalMAC(const SceNetEtherAddr * addr) {
 	return (match == 0);
 }
 
-int isPDPPortInUse(uint16_t port) {
+bool isPDPPortInUse(uint16_t port) {
 	// Iterate Elements
-	int i = 0; for (; i < 255; i++) if (pdp[i] != NULL && pdp[i]->lport == port) return 1;
+	for (int i = 0; i < 255; i++) if (pdp[i] != NULL && pdp[i]->lport == port) return true;
 
 	// Unused Port
-	return 0;
+	return false;
 }
 
-int isPTPPortInUse(uint16_t port) {
+bool isPTPPortInUse(uint16_t port) {
 	// Iterate Sockets
-	int i = 0; for(; i < 255; i++) if(ptp[i] != NULL && ptp[i]->lport == port) return 1;
+	for(int i = 0; i < 255; i++) if(ptp[i] != NULL && ptp[i]->lport == port) return true;
 	
 	// Unused Port
-	return 0;
+	return false;
+}
+
+char* mac2str(SceNetEtherAddr* mac) {
```

Comment:
Ah i already replace this function with a new one, returning std::string instead of char* so it's no longer platform specific and thread-safe